runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -6687,9 +6685,9 @@ makeMemCpyByteToCharBndchkGraph(TR::Compilation *c, int32_t ctrl)
 
 
 
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
+
+
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there's no need to change these lines. May I ask you to change them back to the following?

```
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
```